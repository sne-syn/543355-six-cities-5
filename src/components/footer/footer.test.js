import Footer from './footer';
import React from 'react';
import renderer from 'react-test-renderer';
import {MemoryRouter} from 'react-router-dom';

test(`Render Footer`, () => {
  const tree = renderer
    .create(
        <MemoryRouter>
          <Footer/>
        </MemoryRouter>
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
