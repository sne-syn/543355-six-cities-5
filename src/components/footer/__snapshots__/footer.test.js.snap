// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Footer 1`] = `
<footer
  className="footer container"
>
  <Link
    className="footer__logo-link"
    to="/"
  >
    <img
      alt="6 cities logo"
      className="footer__logo"
      height="33"
      src="img/logo.svg"
      width="64"
    />
  </Link>
</footer>
`;
